examples/data_ops/10_data_ops_intro.py

```diff
@@ -0,0 +1,177 @@
+"""
+
+Introduction to machine learning pipelines with skrub DataOps
```

Comment:
```suggestion
Introduction to machine-learning pipelines with skrub DataOps
```

Composite adjective: here "machine learning" is an adjective, so it is written with a hyphen (OK, it's a nitpick)